Maple2.Server.Game/PacketHandlers/QuestHandler.cs

```diff
@@ -157,4 +177,10 @@ private enum Command : byte {
 
         session.Quest.Complete(quest);
     }
+
+    private static void HandleSkyFortressTeleport(GameSession session) {
+        session.Send(session.PrepareField(2000422, -1, session.CharacterId)
```

Comment:
Let's use the constant for it.
`FameContentsSkyFortressMapID1`

though, I do think `FameContentsSkyFortressMapID0` would be used if Sky Fortress questline hasn't started. I could be wrong though.